scripts/release/debian/deb_install.sh

```diff
@@ -81,13 +82,17 @@ setup() {
             exit 1
         fi
     fi
-    echo "deb [arch=$(dpkg --print-architecture)] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" \
-        > /etc/apt/sources.list.d/azure-cli.list
+    echo "Types: deb
+URIs: https://packages.microsoft.com/repos/azure-cli/
+Suites: ${CLI_REPO}
+Components: main
+Architectures: $(dpkg --print-architecture)
+Signed-by: /etc/apt/keyrings/microsoft.gpg" | sudo tee /etc/apt/sources.list.d/azure-cli.sources
```

Comment:
```suggestion
Signed-by: /etc/apt/keyrings/microsoft.gpg" | tee /etc/apt/sources.list.d/azure-cli.sources
```
`sudo` is redundant since the script is already invoked with `sudo`.
PS: The `sudo` package is not installed in the Docker image, making it somewhat inconvenient to run commands within the image.